### PR TITLE
Update API_reference doc

### PR DESCRIPTION
**Pull request by wanghouqi**
Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

**Review by syan10 — APPROVED** [review]

**Comment by yang8621** [comment]
seems we have combined timestamp and nonce, right?

**Comment by yang8621** [comment]
params["timestamp"] = str(int(time.time() * 1000))

File: docs/API_Reference.md [hunk 1]
```diff
@@ -20,13 +20,14 @@ Currently, the eHSM-KMS-Service now provides the following restful APIs to the c
 
 ## Common Prameters
 This section describes the parameters that are common to all API requests and responses.
+
  | Name | Type | Reference Value | Description |
  |:-----------|:-----------|:-----------|:-----------|
  | appid | string | 12345678-0123-4567-*** | An unique id to request ehsm in a domain, which is requested<br> from ehsm service maintainer |
  | nonce | string  | 2866307081280200*** | 16 bytes random number, 15 minute validity; <br>Taking the current time as the benchmark, detect nonce within 15 minutes, which cannot be repeated  |
  | timestamp | string  | 20211215*** | The timestamp of sending request; format: yyyymmddhhmmssmmm |
```
Comment (yang8621) [inline docs/API_Reference.md:28]: here nonce can be removed.  timestamp should be changed the wordings.

Comment (wanghouqi) [reply docs/API_Reference.md:28]: It will be change.

**Comment by wanghouqi** [comment]
> seems we have combined timestamp and nonce, right?

yes. noces is removed.

File: docs/API_Reference.md [hunk 2]
```diff
@@ -50,51 +50,531 @@ Create a customer master key(CMK) for the user, which can be a symmetric or an a
...
+  | code | int | 200 | The result of the method call, 200 is success, others are fail. |
+  | message | String | "success" | The description of result. |
+  | ciphertext_base64 | String | "uSDos6NLWNVp4sQZS2+mzLvDw***" |The result in json object for the Ciphertext  which in based64 encoding. |
+
+- **Example**
+	- Request sample in python
+  ```python
+    params = OrderedDict()
+    params["appid"] = appid
+    params["nonce"] = str(int.from_bytes(os.urandom(16), "big"))
```
Comment (yang8621) [inline docs/API_Reference.md:115]: nonce can be removed and the timestamp should also be updated.
Update for all the API examples.

Comment (wanghouqi) [reply docs/API_Reference.md:115]: It will be fix.

**Comment by wanghouqi** [comment]
> seems we have combined timestamp and nonce, right?

Yes,It will be change.

**Review by yang8621 — APPROVED** [review]

**Review by syan10 — APPROVED** [review]

**Pull request merged**